application/actions/requests_test.go

```diff
@@ -838,3 +839,141 @@ func (as *ActionSuite) Test_RequestActions() {
 		})
 	}
 }
+
+func (as *ActionSuite) Test_convertRequest() {
+	userFixtures := test.CreateUserFixtures(as.DB, 2)
+	creator := userFixtures.Users[0]
+	provider := userFixtures.Users[1]
+
+	requestFixtures := test.CreateRequestFixtures(as.DB, 2, false, creator.ID)
+
+	meeting := test.CreateMeetingFixtures(as.DB, 1, creator)[0]
+
+	ctx := test.CtxWithUser(creator)
+
+	min := requestFixtures[0]
+	min.ProviderID = nulls.Int{}
+	min.Description = nulls.String{}
+	min.OriginID = nulls.Int{}
+	min.NeededBefore = nulls.Time{}
+	min.Kilograms = nulls.Float64{}
+	min.URL = nulls.String{}
+	min.FileID = nulls.Int{}
+	min.MeetingID = nulls.Int{}
+
+	full := requestFixtures[1]
+	full.ProviderID = nulls.NewInt(provider.ID)
+	full.FileID = nulls.NewInt(test.CreateFileFixture(as.DB).ID)
+	full.MeetingID = nulls.NewInt(meeting.ID)
+
+	tests := []struct {
+		name    string
+		request models.Request
+		want    api.Request
+		wantErr bool
```

Comment:
It doesn't seem like these fields are being used in the tests in Test_convertRequest()

Reply:
Good point